easybuild/easyconfigs/r/RSEM/RSEM-1.2.30-foss-2016a.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'ConfigureMake'
+
+name = 'RSEM'
+version = '1.2.30'
+
+homepage = 'http://deweylab.github.io/RSEM/'
+description = """RNA-Seq by Expectation-Maximization"""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['https://github.com/deweylab/RSEM/archive/']
+sources = ['v%(version)s.tar.gz']
+
+#patches = ['RSEM-%(version)s_zlib.patch']
+patches = ['RSEM-%(version)s_makefiles.patch']
+
+skipsteps = ['configure']
+installopts = "prefix=%(installdir)s"
+
+dependencies = [
+    ('ncurses', '6.0'),
+    ('zlib', '1.2.8'),
+]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['bin/samtools-1.3'],
```

Comment:
hmm, this looks weird... no `rsem-*` binaries?

you don't need to list all of them, just a handful of (key) binaries is sufficient